src/pages/Components/Select/code.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import the settings and the select `scss` files.
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.select';
+```
+
+## Basic usage
+
+Apply the classes `mc-select`on the select wrapper, then `mc-select__input` on the native element of the select and `mc-select__icon` on the arrow's icon svg, as in the following example :
+
+```html
+<div class="mc-select">
```

Comment:
I'm really not sure about that, can't we customize a native select without wrapping it and adding the icon manually to the Dom ? 
We used background B64 images for checkboxes, I think this can be done the same way.